pom.xml

```diff
@@ -1,298 +1,308 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
The PR still has the problem that a whole file marked changed. It might be due to line endings.

Reply:
I have changed line endings from unix to windows for whole project.